lms/djangoapps/mobile_api/utils.py

```diff
@@ -1,14 +1,13 @@
 """
 Common utility methods and decorators for Mobile APIs.
 """
-
-
-import functools
-
 from opaque_keys.edx.keys import CourseKey
 from courseware.courses import get_course_with_access
 from rest_framework import permissions
 from rest_framework.authentication import OAuth2Authentication, SessionAuthentication
+import functools
+import json
+import urllib2
```

Comment:
By the way, per Python conventions, modules are generally imported in a certain order: system imports first, 3rd party libraries, shared libraries, then finally local imports.